.gitignore

```diff
@@ -205,3 +205,7 @@ pg_data
 .ruby-version
 mysql2psql.yml
 *.pem
+
+
+Gemfile.lock
+schema.rb
```

Comment:
Actually, you should not put `schema.rb` in `.gitignore` file since we need to control the different versions of db schema.